location_extractor/clients.py

```diff
@@ -0,0 +1,252 @@
+import os
+import sqlite3
+from dataclasses import dataclass
+from sqlite3 import Cursor, Connection
+from typing import List, Tuple, Union, Optional
+
+from location_extractor import src_dir
+from location_extractor.helpers import remove_accents
+import csv
+
+
+@dataclass(frozen=True, order=True)
+class LocationDTO:
+    locale_code: str
+    continent_code: str
+    continent_name: str
+    country_iso_code: str
+    country_name: str
+    subdivision_name: str
+    city_name: str
+    is_in_european_union: bool
+
+
+class DBClient:
+    def __init__(self) -> None:
+        self.dbpath = os.path.join(src_dir, "data", "data.db")
+
+        self.index_columns = [
+            'continent_name',
+            'country_iso_code',
+            'country_name',
+            'subdivision_name',
+            'city_name',
+        ]
+
+        with self.connect() as conn:
+            table_exists = conn.execute("""
+                SELECT name FROM sqlite_master WHERE type='table' AND name='locations';
+            """).fetchone()
+
+            if not table_exists:
+                conn.execute('''
```

Comment:
let's decide for one way of defining strings and multiline strings. My rules for that are:

+ use `'` (single quote) whenever it's possible (so always except 1 case when string contains `'`)
+ use `'''` (3 x single quote) always except docstring and cases when string contains `'''`

It's good to be consistent

Reply:
I will add `flake8-quotes` in PR with linters to force us to be consistent when it come to that